ansible/roles/anitya-dev/tasks/main.yml

```
@@ -37,7 +37,7 @@
   become_user: "{{ ansible_env.SUDO_USER }}"
   shell: >
       source ~/.bashrc && mkvirtualenv anitya -a ~/devel/ &&
-      virtualenv-3 --system-site-packages ~/.virtualenvs/anitya/
+      virtualenv --system-site-packages ~/.virtualenvs/anitya/
```

Since this was explicitly Python 3 before, it would be good to remain explicit:

```suggestion
      virtualenv --python=/usr/bin/python3 --system-site-packages ~/.virtualenvs/anitya/
```

---

Applied